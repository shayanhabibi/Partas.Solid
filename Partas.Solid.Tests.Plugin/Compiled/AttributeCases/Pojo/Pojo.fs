module Partas.Solid.Tests.AttributeCases.Pojo

open Partas.Solid
open Fable.Core

[<JS.Pojo>]
type Poj(?testAttr: string, ?otherAttr: string) =
    member val testAttr: string = JS.undefined with get, set
    member val testGrok: string = JS.undefined with get, set
    member val testBok: string = JS.undefined with get, set

[<SolidComponent>]
let Bock () =
    let tog = Poj (testGrok = "test", testBok = "bok", testAttr = "chok")
    let bop = Poj (testAttr = "tes")
    let chog = Poj (testGrok = "dfs", otherAttr = "sdf")
    Poj (testGrok = "dfs", otherAttr = "sdf")
