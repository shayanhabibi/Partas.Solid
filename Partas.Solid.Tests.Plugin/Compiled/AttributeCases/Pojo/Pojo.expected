export function Bock() {
    const tog = {
        testAttr: "chok",
        testBok: "bok",
        testGrok: "test",
    };
    const bop = {
        testAttr: "tes",
    };
    const chog = {
        otherAttr: "sdf",
        testGrok: "dfs",
    };
    return {
        otherAttr: "sdf",
        testGrok: "dfs",
    };
}